scenes/game/game.gd

```diff
@@ -523,7 +523,10 @@ func _process(delta):
 		var events = game_wrapper.poll_for_events()
 		if events.size() > 0:
 			_handle_events(events)
-			$CombatLog.set_text(game_wrapper.get_combat_log())
+			if $CombatLog.use_old_log:
```

Comment:
can you go ahead and define this with an @onready var at the top and use that instead of $CombatLog?